src/core_modules/capture-core/components/Pages/MainPage/mainPage.types.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+type PassOnProps = $ReadOnly<{
+    programId: string,
+}>;
+
+export type Props = $ReadOnly<{
+    ...PassOnProps,
+    currentSelectionsComplete: boolean,
+    classes: Object,
+}>;
```

Comment:
Just keeping track of things. Feel free to be more pragmatic and skip my suggestions